guacamole_sg.tf

```
@@ -0,0 +1,114 @@
+# Security group for guacamole instances
+resource "aws_security_group" "guacamole" {
+  provider = aws.provisionassessment
+
+  vpc_id = aws_vpc.assessment.id
+
+  tags = merge(
+    var.tags,
+    {
+      "Name" = "Desktop gateway (Guacamole)"
```

I think this way better matches the style you have elsewhere.
```suggestion
      "Name" = "Guacamole (desktop gateway)"
```